src/get-started/java-binding.md

```diff
@@ -503,6 +503,12 @@ Currently Java Binding includes the following built-in services:
   See the [*Anchoring Service*](../advanced/bitcoin-anchoring.md)
   article for more details.
 
+- **Time Oracle.**
+  Time oracle allows user services to access the calendar time supplied by
+  validator nodes to the blockchain.
```

Comment:
Time Oracle allows user services to access the calendar time which validator nodes supply to the blockchain from the external world.